### ISSUE TITLE
Prevent / discourage overwrites !

### ISSUE THREAD

**Issue by kjcole** [mjsir911/ConfigMaker#26]
I'm not sure how it was done, but apparently, while demoing your code, someone overwrote ALL of the ratings.   Sadly, I did not witness this, and cannot verify how it was done. The speculation is that someone chose the name "ratings" for the set of ratings.  Let's make that a no-no. Ditto for "presets".  (But they would have needed to go up from the default directory too, right?)

And add some sort of warning "Are you SURE???" message when there is a name conflict.

**Comment by mjsir911** [comment]
Yea the code does delete the contents of the directory. you will find the deleted contents in a directory in /tmp or wherever mac puts them

Sent from Alto
On Thursday, August 31, 2017 at 12:33 PM Kevin Cole <notifications@github.com> wrote:

I'm not sure how it was done, but apparently, while demoing your code, someone overwrote ALL of the ratings. Sadly, I did not witness this, and cannot verify how it was done. The speculation is that someone chose the name "ratings" for the set of ratings. Let's make that a no-no. Ditto for "presets". (But they would have needed to go up from the default directory too, right?)

And add some sort of warning "Are you SURE???" message when there is a name conflict.

**Comment by kjcole** [comment]
Nope. I looked. Nothing in ~/.Trash/ ~/tmp/ or /tmp/ that I could find. I searched as root from /. No joy.  Fortunately, on my laptop, I have backups of the most critical files and restored them -- and it's not too difficult to recreate the two main ones we'd been using, from scratch. But it was embarrassing in a live demo.
